transforms/6-change-side/index.ts

```diff
@@ -0,0 +1,30 @@
+import {API, FileInfo, Options} from 'jscodeshift';
+
+import {remapJSXProps} from '../helpers';
+
+const config = {
+    side: {
+        to: 'side',
+        values: {
+            left: 'start',
+            right: 'end',
+        },
+    },
+};
+
+export default function (file: FileInfo, api: API, options: Options) {
+    const j = api.jscodeshift;
+    const root = j(file.source);
+
+    if (
+        !Math.max(
+            Number(remapJSXProps('Button.Icon', config, root, j)),
+            Number(remapJSXProps('ButtonIcon', config, root, j)),
```

Comment:
ButtonIcon is not exported from uikit

Reply:
Then why is it indicated in the issue?
https://github.com/gravity-ui/uikit-codemod/issues/3

Reply:
By accident. Nevermind, let's keep it.